refextract/references/kbs/report-numbers.kb

```diff
@@ -446,3 +503,271 @@ MPP           ---MPP
 NIKHEF        ---NIKHEF
 RAL TR        ---RAL-TR
 SLS TME TA    ---SLS-TME-TA
+
+*****UMI different format*****
+<s99999sMC>
+<s99999>
+
+UMI NQ        ---UMI-NQ
+UMI NR        ---UMI-NR
+UMI MR        ---UMI-MR
+UMI MQ        ---UMI-MQ
+UMI NA        ---UMI-NA
+
+*****UMI*****
+<s99s99999sMC>
+<s999s99999sX>
+<s9s999s99999s9>
+<s999s99999s9>
+<s99s99999?9?>
+
+UMI           ---UMI
+
+
+*****Brookhaven*****
+<s999?9?9?9?>
+<s999999?syyyysJA>
+<s999999?syyyysCP>
+<s999999?syyyysIR>
+<s999999?syyyysJAM>
+
+BNL              ---BNL
+BNL AADD         ---BNL-AADD
+BNL AD RHIC      ---BNL-AD-RHIC
+
+*****Brookhaven different format*****
+<syys99?>
+
+BNL HET          ---BNL-HET
+BNL NT           ---BNL-NT
+BNL CRISP        ---BNL-CRISP
+
+*****MIT*****
+<s999?9?9?>
+<syys999>
+MIT CTP            ---MIT-CTP
+MIT LNS            ---MIT-LNS
+
+****IHEP*****
+<syys99?9?>
+
+IFVE               ---IFVE
+
+*****Los Alamos*****
+<syys99?9?9?>
+
+LA UR              ---LA-UR
+
+*****Los Alamos different format*****
+<s9999?9?sMS>
+<s9999?9?sPR>
+<s9999?9?s[TCM]>
+
+LA                ---LA
+
+*****Valencia IFIC*****
+<syys99?9?9?>
+
+IFIC              ---IFIC
+
+
+*****CalTech*****
+<syyyys99?9?9?>
+<s69s99?99>
+
+CALT             ---CALT
+CALT TH          ---CALT-TH
+
+*****Yukawa Institute*****
+<syys99?9?>
+
+YITP             ---YITP
+YITP SB          ---YITP-SB
+YITP U           ---YITP-U
+
+*****Yukawa Institute different notation*****
+<s999?9?>
+
+YITP K           ---YITP-K
+
+*****TRIUMF*****
+<syys99?9?A>
+<syys99?9?>
+
+TRI PP            ---TRI-PP
+
+*****Durham U. IPPP*****
+<syys99?9?>
+
+IPPP              ---IPPP
+
+
+*****ITEP*****
+<s99?9?syy>
+<syyyys999?>
+
+ITEP TH           ---ITEP-TH
+ITEP LAT          ---ITEP-LAT
+ITEP PH           ---ITEP-PH
+ITEP M            ---ITEP-M
+ITEP YM           ---ITEP-YM
+ITEP              ---ITEP
+
+*****Korea Inst. Advanced Study*****
+<99999?9?>
+
+KIAS P          ---KIAS-P
+KIAS Q          ---KIAS-Q
+
+*****Tokyo Inst. Tech.*****
+<s999?>
+<s999?-NP>
```

Comment:
not sure about the hyphen here, I don't think it'll match

Reply:
I don't think either, as it should have been stripped